x-pack/plugins/security_solution/common/api/asset_criticality/status.yaml

```diff
@@ -0,0 +1,26 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
```

Comment:
I believe version is supposed to match the versioned routes api, so it should most likely be "2023-10-31"  to align with the other versions or maybe "2023-11-30"?

Also, the file should be renamed `status.schema.yaml` to take advantage of the types codegen